tools/sigh.ts

```diff
@@ -635,6 +635,10 @@ function buildWasmModule(emsdk, counts: WasmCounts, configFile: string, logCmd:
       continue;
     }
 
+    if (!fs.existsSync(cfg.outdir)) {
+      fs.mkdirSync(cfg.outdir);
+    }
```

Comment:
Please move this to line 607, just after the `if (cfg.outdir === '$here')` block, add the `recursive` flag and drop the 'exists' check (we don't need it when recursive path creation is on):

`fs.mkdirSync(cfg.outdir, {recursive: true});`

Reply:
Updated!